### DISCUSSION TITLE
23.10.12 데일리 스크럼

### DISCUSSION THREAD

**Discussion by Jeonwoojae** [petit-a-petit/MIML#29] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 피드백 받은 내용에 대한 답변 & 수정
- 내일은 어떤 일을 하나요?
  - @transactioneventlistener 적용

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 코드리뷰
* 내일은 어떤 일을 하나요?
  * 구매 검증로직 조사
  * 환불 정책 및 부분환불 기능 조사

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - PR 완료
    - JWT 발급하지 않고 사용자 검증 방법 고민 (spotify에서 제공하는 액세스 토큰 검증 엔드포인트?)
- 내일은 어떤 일을 하나요?
    - 코드 리뷰 반영

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 플레이리스트 CRUD 테스트 작성 및 구현
- 내일은 어떤 일을 하나요?
    - 플레이리스트 CRUD 테스트 작성 및 구현
    - PR
